fix: DIA-1896: Complete LSDK export tests

robot-ci-heartex opened the pull request:
Hi @niklub!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/409 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13145360986)

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13145360986

Commit:
Update tests

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13200697607)
> Successfully merged:  25 files changed, 415 insertions(+), 127 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-1896'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13200697607

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13201620134

niklub committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13205870089
